packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -1032,13 +1043,16 @@ Future<T> showCupertinoModalPopup<T>({
   required BuildContext context,
   required WidgetBuilder builder,
   ImageFilter? filter,
+  Color barrierColor = _kModalBarrierColor,
```

Comment:
Slightly out of scope but since we're here, let's also make the showCupertinoDialog parameters consistent with these new ones as well

Reply:
Thought I would do a different PR for that since it doesn't involve the popup and I would have to write tests for that too. Would you rather I do that in this PR?

Reply:
Nope. A different PR sounds good :)